models/src/it/java/io/sphere/sdk/reviews/ReviewIntegrationTest.java

```diff
@@ -0,0 +1,392 @@
+import io.sphere.sdk.queries.Predicate;
+import io.sphere.sdk.reviews.Review;
+import io.sphere.sdk.reviews.ReviewDraft;
+import io.sphere.sdk.reviews.commands.ReviewCreateCommand;
+import io.sphere.sdk.reviews.commands.ReviewUpdateCommand;
+import io.sphere.sdk.reviews.commands.updateCommands.SetAuthorName;
+import io.sphere.sdk.reviews.commands.updateCommands.SetScore;
+import io.sphere.sdk.reviews.commands.updateCommands.SetText;
+import io.sphere.sdk.reviews.commands.updateCommands.SetTitle;
+import io.sphere.sdk.reviews.queries.ReviewQuery;
+import io.sphere.sdk.test.IntegrationTest;
+import org.junit.Test;
+
+import java.time.Instant;
+import java.time.format.DateTimeFormatter;
+import java.util.Arrays;
+import java.util.Optional;
+
+import static io.sphere.sdk.customers.CustomerFixtures.withCustomer;
+import static io.sphere.sdk.products.ProductFixtures.withProduct;
+import static io.sphere.sdk.reviews.ReviewsFixtures.withReview;
+import static org.junit.Assert.*;
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Created by njacinto on 17/04/15.
+ */
+
+public class ReviewIntegrationTest extends IntegrationTest {
+    private static final long SERVER_TIME_TOLERANCE_SEC = 120;
+    // { authorName, title, text, score }
+    private static final Object[][] REVIEWS = {
+            {"First author","First review","This is my first comment",0.3f},
+            {"Second author","Second review","This is my second comment",0.4f},
+            {"Third author","Third review","This is my third comment",0.5f},
+            {"Fourth author","Fourth review","This is my fourth comment",0.6f},
+            {"Fifth author","Fifth review","This is my fifth comment",0.7f}
+    };
+
+    @Test
+    public void testCreate() throws Exception {
+        withProduct(client(), product ->
+            withCustomer(client(), customer -> {
+                Instant start = Instant.now().minusSeconds(SERVER_TIME_TOLERANCE_SEC); // time tolerance
+                Review review = client().execute(ReviewCreateCommand.of(ReviewDraft.of(product.getId(), customer.getId())
+                        .withAuthorName((String) REVIEWS[0][0]).withTitle((String) REVIEWS[0][1])
+                        .withText((String) REVIEWS[0][2]).withScore((Float) REVIEWS[0][3])));
+                Instant afterCreation = Instant.now().plusSeconds(SERVER_TIME_TOLERANCE_SEC); // time tolerance
+                assertEquals(product.getId(), review.getProductId());
+                assertEquals(customer.getId(), review.getCustomerId());
+                assertEquals((String) REVIEWS[0][0], review.getAuthorName().get());
+                assertEquals((String) REVIEWS[0][1], review.getTitle().get());
+                assertEquals((String) REVIEWS[0][2], review.getText().get());
+                assertEquals((Float) REVIEWS[0][3], review.getScore().get());
+                assertNotNull(review.getId());
+                assertEquals(1, review.getVersion());
```

Comment:
ReviewImpl extends class io.sphere.sdk.models.DefaultModelImpl which contains the fields `final String id, final long version, final Instant createdAt, final Instant lastModifiedAt`, which are already tested, so you don't need to worry about the fields since they are tested elsewhere.